### ISSUE TITLE
Command-M does not minimize in OS X app

### ISSUE THREAD

**Issue by raine** [gitterHQ/gitter#879]
It doesn't seem to do anything. So I'm wondering why isn't <kbd>⌘-M</kbd> minimize as one would expect?

**Comment by rolandjitsu** [comment]
I get the same issue here :+1:

**Comment by hdsantos89** [comment]
Yes, it doesn't work for me either. I have been googling about it and thought of even doing a fluid app worst case scenario.
Would be great to see a change :)

**Comment by suprememoocow** [comment]
@raine @rolandjitsu @hdsantos89 

I think this was down to me having overridden the default on my development machine. When I created the menu, XCode used *my* settings, not the standard shortcut for Minimize.

Would you mind installing the Beta app from http://update-beta.gitter.im/osx/GitterBeta-1.166.dmg and letting me know if you're still experiencing this problem?

Thanks
Andrew

**Comment by hdsantos89** [comment]
Hey @suprememoocow , I have installed the Beta app and I am still getting the same issue :( Not exactly sure why.

**Comment by suprememoocow** [comment]
Thanks @hdsantos89 would you mind checking what the shortcut key is set to in the the Window menu in Gitter Beta?

**Comment by hdsantos89** [comment]
Here you go :) http://puu.sh/pk3OY/2522dbe6b4.png

**Comment by suprememoocow** [comment]
Thanks so much @hdsantos89! I think I know what the problem is. I'll let you know when I've managed to sort it out

**Comment by suprememoocow** [comment]
Hi @hdsantos89 

Okay, I think this problem has been sorted. Would you mind downloading this latest beta and letting me know if it's fixed on your side.

https://s3.amazonaws.com/update-beta.gitter.im/osx/GitterBeta-1.168.dmg

Thanks once again,

**Comment by v3ctorman** [comment]
Hello! 
Last version 1.177 - same problem here.
https://i.imgur.com/QFcCGXt.png
In fact, the shortcut is there, but you can see how complicated it is:)

**Comment by marijane** [comment]
The fact that command-M does not minimize the app on MacOS may be the single most frustrating thing about using the Gitter app.  It's a shortcut I use on daily basis with most apps I use regularly.  I do not consider the complicated "shortcut" mentioned by v3ctorman to be a true shortcut as it requires five keys and two hands -- a very puzzling design decision.  This is something users should be able to do with one hand, and with the same shortcut used by the vast majority of applications that runs on MacOS.

**Comment by MadLittleMods** [comment]
@v3ctorman @marijane You can download the NW.js version of the macOS app which seems to work just fine with `cmd + m` for minimzing, https://update.gitter.im/osx/Gitter-3.1.0.dmg ([Amazon S3 is down atm so the link won't work until that is resolved](https://status.aws.amazon.com/))
